sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -126,7 +127,8 @@ def _send_xo_txn(self, name, action, space=""):
             dependencies=[],
             payload_encoding="csv-utf8",
             payload_sha512=_sha512(payload),
-            batcher_pubkey=self._public_key
+            batcher_pubkey=self._public_key,
+            nonce=time.time().hex().encode()
```

Comment:
Ok, if we want to change the approach (I did copy this from intkey), then let's create a JIRA for it.